packages/buidler-core/test/internal/buidler-evm/stack-traces/test.ts

```diff
@@ -126,17 +130,36 @@ function defineDirTests(dirPath: string) {
           ? testDefinition.description
           : "Should give the right stack trace";
 
+      const compilerOptions = {
+        solidityVersion,
+        compilerPath,
+        withOptimizations: false,
+        runs: 200,
+      };
+
       const func = async function () {
-        await runTest(dirPath, testDefinition, sources);
+        await runTest(dirPath, testDefinition, sources, compilerOptions);
       };
 
-      const funcWithOptimizations = async function () {
-        await runTest(dirPath, testDefinition, sources, true);
+      const funcWithFewRuns = async function () {
+        await runTest(dirPath, testDefinition, sources, {
+          ...compilerOptions,
+          withOptimizations: true,
+          runs: 200,
```

Comment:
I'd also include `runs: 1`, as it's a commonly used special case. It optimizes the code to have the smallest deployment bytecode that solc can emit.